packages/replay/test/integration/rateLimiting.test.ts

```diff
@@ -64,120 +64,9 @@ describe('Integration | rate-limiting behaviour', () => {
     replay && replay.stop();
   });
 
-  it.each([
```

Comment:
IMO we don't need these tests anymore, as they only tested that we pause correctly for different rate limit timeouts we might receive from the Sentry backend. Just ensuring that we don't do anything after a 429 response should be good enough

Reply:
Yes, makes sense! 👍 We could even move it to the general `sendReplay` tests I guess?

Reply:
Generally I think it makes sense but given that we have this neat comment under the last test,

```js
// NOTE: If you add a test after the last one, make sure to adjust the test setup
// As this ends with a `stopped()` replay, which may prevent future tests from working
// Sadly, fixing this turned out to be much more annoying than expected, so leaving this warning here for now
```
and I ran exactly into this problem, wasting 30minutes trying fix it, I'd vote we leave it as is.

Reply:
ahh, right 🙈 damn we should fix those xD